index.md

```diff
@@ -7,86 +7,51 @@ title: MDAnalysis
-
-## Participating ##
-
-Ask **questions** on the [{{ site.mailinglists.discussion.name }}
-forum]({{ site.mailinglists.discussion.url }}) and **be part of the
-conversation**. You can also join the [{{ site.discord.name }}
-Discord Server]({{ site.discord.url }}) to talk with other users and
-developers. (In order to join our Discord server, use the invitation link
-[{{ site.discord.invite }}]({{ site.discord.invite }}).)
-
-Please report **bugs** or **enhancement requests** through the [Issue
-Tracker]({{ site.github.issues }}). 
-
-MDAnalysis is **open source** and welcomes *your* contributions. [Fork
-the repository on
-GitHub](https://github.com/MDAnalysis/mdanalysis#fork-destination-box)
-and submit a pull request. Participate on the [{{
-site.mailinglists.developer.name }}
-forum]({{ site.mailinglists.developer.url }}).
-
-MDAnalysis regularly takes part in various **mentoring and outreach programs**, such as [Google Summer of Code](https://summerofcode.withgoogle.com/), [Google Season of Docs](https://developers.google.com/season-of-docs), [Outreachy](https://www.outreachy.org/), the [Station1 Frontiers Fellowship](https://www.station1.org/sff), and the [CompChemURG](https://www.bindingsites.co.uk/home) mentorship schemes. Many of the [current core developers](/about/#mdanalysis-core-developers/) joined the project through these mentoring schemes. The project also frequently offers **teaching workshops**, both online and in-person; recordings are often made available on the [MDAnalysis YouTube channel](https://www.youtube.com/@mdanalysis3040). Follow our [blog]({{ site.baseurl }}/blog), [LinkedIn](https://www.linkedin.com/company/mdanalysis/), [Twitter](https://twitter.com/mdanalysis), and [Bluesky](https://bsky.app/profile/mdanalysis.bsky.social) pages for updates on how to participate.
-
-
-## Supporting ##
-
-If you like MDAnalysis and want to support our mission, please
-consider making a donation to support our efforts.
-
-{{ site.numfocus.donate_button }}
-
-(Donations are made through [our fiscal sponsor][], [NumFOCUS][], which is
-a 501(c)(3) non-profit charity in the United States; as such,
-donations to NumFOCUS are tax-deductible as allowed by law.  As with
-any donation, you should consult with your personal tax adviser or the
-IRS about your particular tax situation.)
+<!-- TODO: Correct link so it links to the specific section of this page [multiple formats]({{site.pypi.docs}}/documentation_pages/coordinates/init.html#id1)
+ -->
+**MDAnalysis** is an open-source Python library for analyzing molecular dynamics (MD) trajectories across [multiple formats]({{site.pypi.docs}}/documentation_pages/coordinates/init.html#id1). It enables seamless reading and writing of simulation data, allowing users to efficiently analyze molecular structures and dynamics, including particle-based trajectories and individual coordinate frames (e.g., biomolecules in the [PDB format][]). 
```

Comment:
I know that this is the rewrite of the original page, but it seems a bit narrow (it was written 10+ years ago...). I feel we should start bigger with the MDAnalysis Project and then mention the MDAnalysis library itself.

Reply:
The second sentence should be "The MDAnalysis Project is a fiscally sponsored project of NF, a ....".